### DISCUSSION TITLE
No puedo entrar al formulario para enviar las katas

### DISCUSSION THREAD

**Discussion by MezaDonovan** [LaunchX-InnovaccionVirtual/CursoIntroPython#331] [Q&A]
Tengo una duda gente :(, al intentar entrar al formulario para enviar las katas me dice que no tengo permisos para responderlo, en ese caso que puedo hacer? :(

**Comment by RonnieRuuz** [comment]
Hola Donovan, ese error aparece cuando ya se tiene otra sesión iniciada. 
Intenta cerrar primero tu sesión en https://www.office.com y luego abrir el link del formulario https://forms.office.com/r/mLLd3dbfPX
Recuerda iniciar con el correo de innovacción.

Espero te haya sido de ayuda.

**Reply by JoseCondorGuere** [reply]
No permite entregar las katas @FernandaOchoa  recuerdo que dijeron que se tendría una semana más para quienes iban a paso lento 😢

**Reply by Gustavo-Valdez** [reply]
Ya lo intente y no se puede, el formulario no esta abierto

**Reply by MezaDonovan** [reply]
Entren en una pestaña de incógnito!
